### DISCUSSION TITLE
Why Forge is way faster than Comfy with the Normal Flux Dev  Version

### DISCUSSION THREAD

**Discussion by void95** [lllyasviel/stable-diffusion-webui-forge#1063] [General]
Hey im just wondering why i get in comparison to comfyui such a speed gain with the normal flux dev model. In Forge i can run it with around 4s/it in comfy it only runs with around 6,5-7s/it. Is it still using NF4 in forge, even if its the normal dev model?

**Comment by a3nima** [comment]
Afaik forge currently only supports the fp8 and nf4 models. The "normal" fp16 models are not supported yet, if you meant these.

**Comment by void95** [comment]
Mh im using the fp8 model or the nf4. When i use forge the benefit of the nf4 is nearly not existent, thats also weird to me. I get around 4s/it with fp8 and 3,5s/it with nf4. In Comfy i get 6,5-7s/it with fp8 and 3,5s/it with nf4. Seems weird to me.

**Comment by bedovyy** [comment]
nf4 tested on RTX3060, forge was about 7sec faster. (70.5sec vs 77.3sec)
In case of RTX3090 and RTX4090, no difference.
maybe different logic of cpu offloading or something.
